src/MultiSelect/MultiSelect.js

```diff
@@ -0,0 +1,30 @@
+import React, { PureComponent } from 'react';
+import Select from 'react-select';
```

Comment:
@nazaninreihani I am not sure, but curious. Is this supposed to be in binary-components? I do not see any other components in next-gen project.

Reply:
No, I just wrote it for next-gen to use it in some other parts of the app too, it would be better if it was in binary-components, this is where you wanted me to implement an HOC, isn't it fine already?

Reply:
I feel this belongs to binary-components. Please submit it over there and I will get it merged.